packages/playground-examples/copy/ja/TypeScript/Primitives/Any.ts

```diff
@@ -0,0 +1,51 @@
+// AnyはTypeScriptのエスケープ句です。 
+// any型を利用することで、一区切りのコードを
+// JavaScriptの様に動的に扱ったり、
+// 型システムの制限を回避することができます。
+
+// any型を利用する良いケースはJSON parsingです
+
+const myObject = JSON.parse("{}");
+
+// TypeScriptにおけるAny宣言は、あなたがその値について詳しく知っていて
+// 安全なものなので信じてください、という宣言です。
+// それが厳密に正しくない場合でも。例えば、次のコードはクラッシュします。
+
+myObject.x.y.z;
+
+// any型を利用することで、型安全のトレードオフと引き換えに、
```

Comment:
IMO: with the trade offで「引き換えに」になるのでトレードオフはなくても良さそう
```suggestion
// any型を利用することで、型安全性と引き換えに、
```